dotCMS/src/integration-test/java/com/dotcms/experiments/business/ExperimentAPIImpIT.java

```diff
@@ -501,6 +539,131 @@ public void pageReachDirectReferer() throws DotDataException, DotSecurityExcepti
+                .limit(1)
+                .findFirst()
+                .orElseThrow(() -> new AssertionError("Must have a not DEFAULT variant"));
+
+        final Variant variant = APILocator.getVariantAPI().get(variantName).orElseThrow();
+
+        final Instant today = Instant.now();
+        final Instant yesterday = today.minus(1, ChronoUnit.DAYS);
+        final Instant tomorrow = today.plus(1, ChronoUnit.DAYS);
+
+        final List<Map<String, String>> todayCubeJsQueryData = createPageViewEvents(today,
+                experiment, variantName, pageA, pageB, pageD, pageC);
+
+        final List<Map<String, String>> yesterdayCubeJsQueryData = createPageViewEvents(yesterday,
+                experiment, variantName, pageA, pageB, pageD, pageC);
+
+        final List<Map<String, String>> tomorrowCubeJsQueryData = createPageViewEvents(tomorrow,
+                experiment, variantName, pageA, pageB, pageD, pageC);
+
+        final List<Map<String, String>> todayCubeJsQueryDataDefault = createPageViewEvents(today,
+                experiment, DEFAULT_VARIANT.name(), pageA, pageB, pageD, pageC);
+
+        final List<Map<String, String>> tomorrowCubeJsQueryDataDefault = createPageViewEvents(tomorrow,
+                experiment, DEFAULT_VARIANT.name(), pageA, pageB, pageC);
+
+        final Map<String, List<Map<String, String>>> cubeJsQueryResult =  map("data",
+                concat(todayCubeJsQueryData, tomorrowCubeJsQueryDataDefault, yesterdayCubeJsQueryData,
+                        tomorrowCubeJsQueryData, todayCubeJsQueryDataDefault)
+        );
+
+        APILocator.getExperimentsAPI()
+                .start(experiment.getIdentifier(), APILocator.systemUser());
+
+        IPUtils.disabledIpPrivateSubnet(true);
+        final String cubeJSQueryExpected = getExpectedPageReachQuery(experiment);
+
+        final MockHttpServer mockhttpServer = new MockHttpServer(CUBEJS_SERVER_IP, CUBEJS_SERVER_PORT);
+
+
+        addContext(mockhttpServer, cubeJSQueryExpected, JsonUtil.getJsonStringFromObject(cubeJsQueryResult));
+
+        final String queryTotalPageViews = getTotalPageViewsQuery(experiment.id().get(), "DEFAULT", variantName);
+        final List<Map<String, Object>> totalPageViewsResponseExpected = list(
+                map("Events.variant", variantName, "Events.count", "12")
+        );
+
+        addContext(mockhttpServer, queryTotalPageViews,
+                JsonUtil.getJsonStringFromObject(map("data", totalPageViewsResponseExpected)));
+
+        mockhttpServer.start();
+
+        try {
+            final AnalyticsHelper mockAnalyticsHelper = mockAnalyticsHelper();
+
+            ExperimentAnalyzerUtil.setAnalyticsHelper(mockAnalyticsHelper);
+
+            final ExperimentsAPIImpl experimentsAPIImpl = new ExperimentsAPIImpl(mockAnalyticsHelper);
+
+            final ExperimentResults experimentResults = experimentsAPIImpl.getResults(experiment);
+
+            mockhttpServer.validate();
+
```

Comment:
Looks like the test assert is missing

Reply:
done https://github.com/dotCMS/core/pull/24625/commits/989229988f13072ff026d0e585bd2ddb61304cc3#diff-25dea0f580007d55e9704544452fcf0833029a2294fb814f4dddfb1bb2e4ba9bR681